### DISCUSSION TITLE
Add section in issues on Github about "current common issues"

### DISCUSSION THREAD

**Discussion by twistios** [fast4x/RiMusic#3059] [Ideas]
In github one can create a "header" for the issue section (I think via pinning of issues? Maybe also custom text).
I think it would be good to pin/add a note about a very common issue at the time (currently the lyrics problem or in the past the Youtube api change), because there are many issues posted because of only a small select amount of problems actually.

**Comment by aneesh1122** [comment]
> In github one can create a \"header\" for the issue section (I think via pinning of issues? Maybe also custom text).

You actually expect people to read it? While creating the issues it's clearly mentioned that you should search the existing issues before creating a new one.

If I was in fast4x place I would have closed every duplicate issues without any explanation 😂😂

**Comment by fast4x** [comment]
I haven't time for this...
